tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,246 @@
+
+To generate the multilocation derivative account, first clone Alberto’s [xcmTools repo](https://github.com/albertov19/xcmTools){target=_blank}. Run `yarn` to install the necessary packages and then run:
+
+
+    ts-node calculateMultilocationDerivative.ts \
+     --w wss://wss.api.moonbase.moonbeam.network \
+     --a YOUR-MOONBASE-RELAY-ACCOUNT-HERE \
+     --n 0x57657374656e64
+
+Let's review the parameters passed along with this command:
+
+* The `-w` flag corresponds to the endpoint we’re using to fetch this information
+* The `-a` flag corresponds to your Moonbase relay chain address
+* The `-n` flag corresponds to the encoded form of “westend”, the name of the relay chain that Moonbase relay is based on
+
+The script will return 32-byte and 20-byte addresses. We’re interested in the ethereum-style account - the 20-byte one. Feel free to look up your multilocation derivative account on [Moonscan](https://moonbase.moonscan.io/){target=_blank}. You’ll note that this account is empty. You’ll now need to fund this account with at least 1.1 DEV. As this is the amount that the faucet dispenses, you'll need to make a minimum of two faucet requests or you can always reach out to us on [Discord](https://discord.com/invite/amTRXQ9ZpW){target=_blank} for additional DEV tokens.
+
+## Preparing to Stake on Moonbase Alpha {: #preparing-to-stake-on-moonbase-alpha }
+
+The following section will walk through fetching collator information via the [Moonbase Alpha Staking dApp](https://apps.moonbeam.network/moonbase-alpha/staking){target=_blank} and the Polkadot.js Apps UI. If you'd prefer to fetch this information programmatically via the Polkadot.js API, you can visit the [following section.](#retrieving-the-list-of-candidates)
+
+First and foremost, you’ll need the address of the collator you want to delegate to. To locate it, head to the [Moonbase Alpha Staking dApp](https://apps.moonbeam.network/moonbase-alpha/staking){target=_blank} in a second window. Ensure you’re on the correct network, then press **Select a Collator**. Next to your desired collator, press the **Copy** icon. You’ll also need to make a note of the number of delegations your collator has. The [PS-31 collator](https://moonbase.subscan.io/account/0x3A7D3048F3CB0391bb44B518e5729f07bCc7A45D){target=_blank} shown below has `60` delegations at the time of writing. 
+
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+You can also programmatically fetch all of the required staking information. Feel free to skip to the [next section](#generating-the-encoded-call-data) if you'd like.
+
+### Retrieve the List of Candidates {: #retrieve-the-list-of-candidates } 
+
+Prior to staking tokens, you’ll need the address of the collator you want to delegate to. To retrieve the list of collator candidates available in the network, head to the **Developer** tab, click on **Chain State**, and take the following steps:
```

Comment:
Just realized that these sections are reused. I think we should probably put them into snippets so if something changes we only have to make the change in one place

Reply:
- ### Retrieve the List of Candidates {: #retrieve-the-list-of-candidates } 
- ### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations } 
- ### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations }

Reply:
Linking to these sections instead of copying them to keep the tutorial as streamlined as possible